### ISSUE TITLE
Change the name to "CIVL AirScore"

### ISSUE THREAD

**Issue by joergewald** [FAI-CIVL/FAI-Airscore#275]
Maybe it's even time to move away from AirScore, since this is a completely different system from Geoff's original AirScore.

But in any case, the "FAI" must go away, it seems almost preposterous here.

**Comment by kuaka** [comment]
I agree. It's confusing for people who know about Geoff's airscore. 
Suggestions anyone?
VL score? 
VL Comp?

**Comment by biuti** [comment]
Joerg proposed CIVL AirScore but as you say we can chose a different name altogether to avoid confusion.

**Comment by philderbeast** [comment]
Whatever this is, it is not airScore anymore. Yes please change the name.

**Comment by philderbeast** [comment]
Here are some suggestions off the top of my head (not all serious):

* Way Points
* Wind Lines
* Wing Loading
* High Times
* Best Glide
* Sink Rating
* Sky Out
* Air Flow
* Trajectory
* Trailing Edge
* Hurry Up
* Air Foil
* Carried Aloft
* Wing Span

**Comment by joergewald** [comment]
I like VL, for Vol Libre, a very nice term that covers both hang-gliding and paragliding, and has a good link to CIVL. 

Vol Libre Scoring, short VLS?

**Comment by joergewald** [comment]
or SIVL, as in ScorIng Vol Libre?

**Comment by biuti** [comment]
VL Score / VLS is not bad in my opinion.

**Comment by philderbeast** [comment]
Sorry but I think the name SVL is already taken by @DanielDimov for his scoring project.

> I spoke with some French pilots and they advised me to make it more understandable to French-speaking flying community - the name will be SVL (Scoring de Vol Libre).

**Comment by joergewald** [comment]
I think the distinction between VLS and SLV is big enough. :)

**Comment by philderbeast** [comment]
If I go to a car yard, the badge is Ford, not "Ford Car". I'd prefer Free Flight over Free Flight Scoring.

**Comment by joergewald** [comment]
That's because "Ford" has been a thing to sit in and drive around with for over a century, so everybody makes the correct association.

We need to establish a new software. We can either pick a fantasy name, and hope that people catch on over time. Or we say exactly what that program does, and offer an abbreviation for people to use once they understand what that program does.

"Free Flight" - or "Vol Libre" - is very generic. Could also be a hang-gliding simulator, an XC planning tool, or a weather forecast application specific to our sports.

**Comment by kuaka** [comment]
> If I go to a car yard, the badge is Ford, not "Ford Car". I'd prefer Free Flight over Free Flight Scoring.

in addition to Joerg's point.
I think that that is probably not a fair comparison as like a lot of brands 'Ford' is a surname and not an existing term within automobiles etc. Vol Libre is a term that is already used in our sports so there could be confusion.

**Comment by kuaka** [comment]
Perhaps there's room for the middle ground. Not a commonly used term in our spot and not an overly descriptive name. Something like airscore. Perhaps this is useful: 
https://namelix.com/
